boilerplate/app/utils/format-date.ts

```diff
@@ -0,0 +1,28 @@
+import { Locale, format, parseISO } from "date-fns"
+import I18n from "i18n-js"
+
+import ar from "date-fns/locale/ar-SA"
+import ko from "date-fns/locale/ko"
+import en from "date-fns/locale/en-US"
+
+interface Options {
```

Comment:
Does date-fns not export this type?

Reply:
The base of this PR is Yulian's so the conflicts might come in there.

I wish! they only export 4 types: 

<img width="345" alt="Screen Shot 2022-08-26 at 2 13 21 PM" src="https://user-images.githubusercontent.com/9324607/186966563-4fa630d0-0208-4ff9-b6c2-07d3b3a4d0fe.png">

Reply:
@mazenchami 

```
Parameters<typeof format>[2] 
```
will get you the type for the `options` arg

Reply:
ummmm TIL!!!